caas/kubernetes/provider/customresourcedefinitions.go

```diff
@@ -26,14 +26,14 @@ import (
 
 //go:generate mockgen -package mocks -destination mocks/crd_getter_mock.go github.com/juju/juju/caas/kubernetes/provider CRDGetterInterface
 
-func (k *kubernetesClient) getCRDLabels(appName string) map[string]string {
+func (k *kubernetesClient) getCRDLabelsGlobal(appName string) map[string]string {
 	return map[string]string{
 		labelApplication: appName,
 		labelModel:       k.namespace,
 	}
 }
 
-func (k *kubernetesClient) getCRLabels(appName string) map[string]string {
+func (k *kubernetesClient) getCRDLabelsNamespaced(appName string) map[string]string {
```

Comment:
These seem to be the wrong way around?
ie getCRDLabelsNamespaced() should contain the model/namespace?

Ah, I think I understand the use case now. But the method names are confusing.
Especially since we use getCRDLabelsGlobal() for CRDs regardless of scope.
We only use these 2 methods for CRs.

Would it be easier simply to always add the app name and mode name labels to all CRs regardless of scope?

Reply:
Here we just keep the labels are consistent for all global resources.
Like cluster scope RBAC resources, we have appName and model-uuid in labels,
But all the namespaced resources don't have model-uuid in labels.

Reply:
probabaly the naming of the func is confusing.
I have renamed it to `getAPIExtensionLabelsNamespaced` because it's used for CRD and CR(CR could be cluster/namespaced scope)